src/components/movieList.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+import "./../styles/movieList.scss";
+import { Fragment } from "react";
+import PropTypes from "prop-types";
+import MovieService from "../service/movieService";
+
+class MovieList extends Component {
+  static propTypes = {
+    list: PropTypes.any,
+  };
+
+  movies = {
+    size: 0,
+    items: [],
+  };
+  getDefaultList() {
```

Comment:
let's try to implement a container component which will provide all necessary data. and leave this component as presentational